secretstores/alicloud/parameterstore/parameterstore_test.go

```diff
@@ -202,3 +202,13 @@ func TestBulkGetSecret(t *testing.T) {
 		})
 	})
 }
+
+func TestGetFeatures(t *testing.T) {
+	m := secretstores.Metadata{}
+	s := NewParameterStore(logger.NewLogger("test"))
+	s.Init(m)
+	t.Run("no features are advertised", func(t *testing.T) {
+		f := s.Features()
+		assert.Equal(t, 0, len(f))
```

Comment:
nit:
```suggestion
		assert.Empty(t, f)
```

Reply:
I am pushing a revision with this change.